gulpfile.js

```diff
@@ -284,7 +284,7 @@ gulp.task('jsdoc', cb => {
  * Test
  */
 
-gulp.task('test', ['test:unit', 'test:a11y']);
+gulp.task('test', ['test:unit']);
```

Comment:
Need to sort out the a11y issues the team hit. Also, if you simply disable a11y test in Travis, you can change .travis.yml instead of Gulp general test task.